benchmarks/bvh_driver/bvh_driver.cpp

```diff
@@ -227,6 +227,8 @@ int main(int argc, char *argv[])
 
   std::cout << "ArborX version: " << ArborX::version() << std::endl;
   std::cout << "ArborX hash   : " << ArborX::gitCommitHash() << std::endl;
+  std::cout << "Kokkos version: " << ArborX::Details::kokkosVersion()
```

Comment:
Please define `KokkosExt::version()` in `ArborX_DetailsKokkosExtVersion.hpp` which you can include from `ArborX_Version.hpp`